JSQMessagesViewController/Views/JSQMessagesTypingIndicatorFooterView.m

```diff
@@ -109,7 +124,123 @@ - (void)configureWithEllipsisColor:(UIColor *)ellipsisColor
     
     [self setNeedsUpdateConstraints];
     
-    self.typingIndicatorImageView.image = [[UIImage jsq_defaultTypingIndicatorImage] jsq_imageMaskedWithColor:ellipsisColor];
+    // Configure correct color and (whether/not) animated
+    [self setTypingIndicatorAnimated:animated :ellipsisColor];
+}
+
+- (void) setTypingIndicatorEllipsisColor:(UIColor*) ellipsisColor
+{
+    self.typingIndicatorView1.backgroundColor = ellipsisColor;
+    self.typingIndicatorView2.backgroundColor = ellipsisColor;
+    self.typingIndicatorView3.backgroundColor = ellipsisColor;
+}
+
+- (void) setTypingIndicatorAnimated:(BOOL) animated :(UIColor*) ellipsisColor
```

Comment:
um... there's no parameter name for `ellipsisColor` ??

formatting style:

```objective-c
- (void)setTypingIndicatorAnimated:(BOOL)animated ellipsisColor:(UIColor *)ellipsisColor
```

let's run through all of these additions/changes and make sure style is consistent with the rest of the project.